configs/knot_raspberrypi3_defconfig

```diff
@@ -63,6 +63,7 @@ BR2_PACKAGE_KNOT_WEB=y
 BR2_PACKAGE_KNOT_FOG=n
 BR2_PACKAGE_KNOT_FOG_CONNECTOR=y
 BR2_PACKAGE_KNOT_BABELTOWER=y
+BR2_PACKAGE_KNOT_STORAGE=y
```

Comment:
I would not add this package on this file.

Reply:
Done.